packages/app-builder/src/models/api-keys.ts

```diff
@@ -10,16 +10,18 @@ function isApiKeyRole(role: string): role is ApiKeyRole {
 
 export interface ApiKey {
   id: string;
-  organizationId: string;
   description: string;
+  organizationId: string;
+  prefix: string;
   role: ApiKeyRole | 'UNKNWON';
 }
 
 export function adaptApiKey(apiKeyDto: ApiKeyDto): ApiKey {
   const apiKey: ApiKey = {
     id: apiKeyDto.id,
-    organizationId: apiKeyDto.organization_id,
     description: apiKeyDto.description,
+    organizationId: apiKeyDto.organization_id,
+    prefix: apiKeyDto.prefix + '*************',
```

Comment:
Either store it as `value` or only set the `prefix` (you can still insert `'*******'` when formating the value)

Both solution are fine to me